schemas/context/locationcontext.schema.json

```diff
@@ -35,5 +35,5 @@
       "$ref": "#/definitions/locationcontext"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
Will be affected by #170 👎

Reply:
We don't have a specific plan to change this right now, and it is needed for our integration with analytics data for 0.9 milestone, so I feel that we need to bite the bullet and move this forward.  There will still be an opportunity to change before we declare it stable.

Reply:
👎 this refers to geo, which isn't stable - so we can't stabilize this.